### ISSUE TITLE
[i18n] [date-picker] How can I change the DatePicker's first day of week?

### ISSUE THREAD

**Issue by martonx** [element-plus/element-plus#10370]
Bug Type: **`i18n`**

## Environment

- Vue Version: `3.2.41`
- Element Plus Version: `2.2.19`
- Browser / OS: `Chrome`
- Build Tool: `Vite`

### Related Component

- `el-date-picker`

### Reproduction Link

[Docs](https://element-plus.org/en-US/guide/i18n.html)

### Steps to reproduce

Set ConfigProvider to any localization (in my example en or hu). The first day of the week remains Sunday. Localization works as all of the calendar texts translating to Hungarian or English. Except first day of the week.

Please advice,
Lajos

### What is Expected?

Hungarian localization should use Monday as first day of week.

### What is actually happening?

First day of week remains Sunday.

### Additional comments

_(empty)_

**Comment by chenxch** [comment]
look #10200

**Comment by martonx** [comment]
Sorry, I can't speak Korean, or Chinese, or Japan, or whatever those strange characters :(
Could you summarize in English, what should set for getting first day of week working correctly?

**Comment by chenxch** [comment]
Option One:

main.js/ts

```
import 'dayjs/locale/hu'
```


Option Two:

[demo](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtY29uZmlnLXByb3ZpZGVyIDpsb2NhbGU9XCJ6aENuXCI+XG4gICAgPGVsLWNhbGVuZGFyIHYtbW9kZWw9XCJ2YWx1ZVwiIC8+XG4gIDwvZWwtY29uZmlnLXByb3ZpZGVyPlxuPC90ZW1wbGF0ZT5cblxuPHNjcmlwdCBsYW5nPVwidHNcIiBzZXR1cD5cbmltcG9ydCB7IHJlZiB9IGZyb20gJ3Z1ZSdcbmltcG9ydCB7IEVsQ29uZmlnUHJvdmlkZXIsIGRheWpzIH0gZnJvbSAnZWxlbWVudC1wbHVzJ1xuaW1wb3J0IHpoQ24gZnJvbSAnZWxlbWVudC1wbHVzL2Rpc3QvbG9jYWxlL3poLWNuLm1qcydcblxuZGF5anMuZW4ud2Vla1N0YXJ0ID0gMVxuY29uc3QgdmFsdWUgPSByZWYobmV3IERhdGUoKSlcbjwvc2NyaXB0PlxuIiwiaW1wb3J0X21hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge31cbn0iLCJfbyI6e319)
You can manually set `weekStart`.

**Comment by chenxch** [comment]
@martonx Hi, has your issue been resolved?

**Comment by chenxch** [comment]
We've updated the documentation for this section in #10603.

**Comment by martonx** [comment]
Hi yes, thank you very much!

**Comment by chenxch** [comment]
> Hi yes, thank you very much!

You're welcome.
